client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,38 +1,147 @@
+import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import CssBaseline from '@material-ui/core/CssBaseline';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import { Box } from '@material-ui/core';
 import useStyles from './useStyles';
-import { useAuth } from '../../context/useAuthContext';
-import { useSocket } from '../../context/useSocketContext';
-import { useHistory } from 'react-router-dom';
-import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
-import { useEffect } from 'react';
+import { Typography } from '@material-ui/core';
+import { Button } from '@material-ui/core';
+import Navigation from './Navigation/Navigation';
+import UserDashInfo from './UserDashInfo/UserDashInfo';
+import ScheduleOption from './ScheduleOption/ScheduleOption';
+import { string } from 'yup/lib/locale';
+// import { useAuth } from '../../context/useAuthContext';
+// import { useSocket } from '../../context/useSocketContext';
+// import { useHistory } from 'react-router-dom';
+// import { useEffect } from 'react';
+// import CircularProgress from '@material-ui/core/CircularProgress';
+// import { Paper } from '@material-ui/core';
 
 export default function Dashboard(): JSX.Element {
+  // const { loggedInUser } = useAuth();
+  // const { initSocket } = useSocket();
+
+  // const history = useHistory();
+
   const classes = useStyles();
+  const dashOptions = ['EVENT TYPES', 'SCHEDULED EVENTS'];
+  const [dashOptionSelected, setDashOption] = React.useState(dashOptions[0]);
+  const schedOptions = ['UPCOMING', 'PENDING', 'PAST'];
+  const [schedSelect, setSchedSelect] = React.useState(schedOptions[0]);
+  const meetingOptions = [15, 30, 45];
+
+  // useEffect(() => {
+  //   initSocket();
+  // }, [initSocket]);
 
-  const { loggedInUser } = useAuth();
-  const { initSocket } = useSocket();
+  // if (loggedInUser === undefined) return <CircularProgress />;
+  // if (!loggedInUser) {
+  //   history.push('/login');
+  //   // loading for a split seconds until history.push works
+  //   return <CircularProgress />;
+  // }
```

Comment:
We should probably revert this  back to working again and then maybe we can figure out a way to have the user signed in without having to comment out a bunch of code?

Reply:
I reverted the comment back, but I replaced `const { loggedInUser } = useAuth();` with `const loggedInUser = mockLoggedInUser;` using `mockLoggedInUser` from `/mocks/mockUser`. I added a comment that this changed needs to be reverted when the authentication or the demo user is complete.